lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1116,7 +1115,14 @@ def setUp(self):
         self.request = RequestFactory().get('/')
         self.request.user = self.user
         self.request.session = {}
-        self.course = CourseFactory.create()
+        self.course = CourseFactory.create(
+            org="A",
+            number="B",
+            run="C",
+        )
+        course_key = SlashSeparatedCourseKey.from_deprecated_string("A/B/C")
```

Comment:
Can't we grab the course_key directly from the `self.course` that we created above?